plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -202,3 +202,16 @@
     comment on column {{ relation }}.{{ adapter.quote(column_name) if column_dict[column_name]['quote'] else column_name }} is {{ escaped_comment }};
   {% endfor %}
 {% endmacro %}
+
+{%- macro postgres__get_show_grant_sql(relation) -%}
+  select grantee, privilege_type
+  from information_schema.role_table_grants
+      where grantor = current_role
+        and grantee != current_role
+        and table_schema = '{{ relation.schema }}'
+        and table_name = '{{ relation.identifier }}'
+{%- endmacro -%}
+
+{% macro postgres__are_grants_copied_over_when_replaced() %}
+    {{ return(False) }}
+{% endmacro %}
```

Comment:
is this used anywhere?

Reply:
@McKnight-42 I think this needs to be renamed to `postgres__copy_grants`! Since we've now renamed `are_grants_copied_over_when_replaced` to `copy_grants`

Reply:
ranemed in this pr and pushed up --Done